composer.json

```diff
@@ -16,19 +16,19 @@
             "name":"Michael Bodnarchuk"
         }
     ],
-    "minimum-stability": "RC",
+    "minimum-stability": "dev",
     "require": {
-        "php": ">=7.4.0 <=8.1 | ~8.1.0",
-        "codeception/lib-innerbrowser": "^1.0",
-        "codeception/codeception": "^4.0"
+        "php": ">=8.1.0",
+        "codeception/codeception": "dev-5.0-interfaces as 5.0.0",
```

Comment:
now we can safely use `"^5.0.0-alpha2",`

Reply:
@developedsoftware You have to identify the module that is causing the conflict and update it there too `"^5.0.0-alpha2"` is the correct restriction.

Reply:
OK done -> https://github.com/Codeception/module-filesystem/pull/18